### DISCUSSION TITLE
[Projects Beta] Iteration field does not start with the selected date.

### DISCUSSION THREAD

**Discussion by jamesros161** [community/community#7789] [Projects and Issues]
When creating an Iteration field, and choosing the start date of the first iteration, if the start date is in the past, it doesn't work. Instead it selects today as the current date of the iteration.  Additionally, there is no way to modify the date / duration of an iteration after it has been created. 
![Iteration_bug](https://user-images.githubusercontent.com/49331357/142041365-87296975-739e-4cd8-a625-8dd83567577f.gif)

**Comment by ryihan** [comment]
Big Problem.....

**Comment by mosaa** [comment]
Need to fix ASAP

**Comment by BigTMiami** [comment]
Just started trying to use projects and ran into this.  It is a problem
